cypress/tests/scroll-to-contest.js

```diff
@@ -13,9 +13,9 @@ describe('Review Page', () => {
     cy.wrap(clickThoughPages).each(() => {
       cy.contains('Next').click()
     })
-    cy.get('#county-commissioners').click()
+    cy.get('#contest-county-commissioners').click()
```

Comment:
Nothing new introduced here, but a note for future test writing… we really need to be selecting things based upon descriptions of things which the voter can find and click on… eg… not element ids.